pyproject.toml

```diff
@@ -190,16 +195,17 @@ ignore = [
     # Errors from function calls in argument defaults. These are fine when the result is immutable.
     "B008",
     # __magic__ methods are are often self-explanatory, allow missing docstrings
-    "D105",
+    #"D105",
     # first line should end with a period [Bug: doesn't work with single-line docstrings]
     "D400",
     # First line should be in imperative mood; try rephrasing
     "D401",
-    ## Disable one in each pair of mutually incompatible rules
-    # We don’t want a blank line before a class docstring
-    "D203",
```

Comment:
Add this again? I don’t have preference here.

Reply:
Please suggest a fix on top and lets see tests finish again